### PR TITLE
fix: handle null values for descriptions and schemas properties

### PR DESCRIPTION
**Pull request by dpopp07**
The validator was crashing on the COS spec due to some properties without values. While these were unintentional, the code should still be able to run fine.

This cleans up the areas blocking a successful run on that specific spec.

### CODE REVIEW

**Comment by mkistler** [comment]
Are properties with "null" schemas acceptable in OpenAPI? If so, this condition should be at least a warning, and if not, there should be an error issued.  Is that the case?

**Comment by dpopp07** [comment]
That's a great question. I was just trying to get the fix out quickly, but we should let the user know if they are adding blank items in their spec. I will look into this and update the PR accordingly.

**Comment by dpopp07** [comment]
@mkistler as far as I can tell, null values are _not_ allowed by the spec. So, I added a non-configurable validation that will return an error if any property in the spec has a null value.

File: README.md [hunk 1]
```diff
@@ -217,6 +217,7 @@ The supported rules are described below:
 | --------------------------- | ----------------------------------------------------------------------------- | -------- |
 | invalid_type_format_pair    | Flag any schema that does not follow the [data type/format rules.][2]         | shared   |
 | snake_case_only             | Flag any property with a `name` that is not lower snake case.                 | shared   |
+| no_schema_description       | Flag any schema without a `description` field.                                | shared   |
```
Comment (mkistler) [inline README.md]: I don't understand what is going on here.  This line is _already_ in `master`, so why is GHE showing this in the diff?

**Comment by dpopp07** [comment]
@mkistler This has been rebased on top of `master` and is ready for review now

**Pull request merged**